src/test/scala/tech/cryptonomic/conseil/tezos/JsonDecodersTest.scala

```diff
@@ -341,7 +341,18 @@ class JsonDecodersTest extends WordSpec with Matchers with EitherValues with Opt
 
       }
 
-      "decode an transaction operation from json" in new OperationsJsonData {
+    "decode an transaction with internal transaction result operation from json" in new OperationsJsonData {
```

Comment:
For me it's fine to remove the other test entirely: I don't think we're supposed to handle differently transactions, based on the presence or absence of internal results.

Less code to maintain 😁

Reply:
I wanted to test if it's correctly deserialized, I think it's needed

Reply:
You got me wrong, I'd keep the one you added now. I'd get rid of the previous one for transaction, this one looks more complete

Reply:
ok, removed, but coverage failed: -0,02% 😄